src/test/java/seedu/address/testutil/TypicalEvents.java

```diff
@@ -35,7 +35,7 @@
             .withTags("meeting").build();
 
     public static final Event FOOTBALL_PRACTICE = new EventBuilder().withName("Football Practice")
-            .withAddress("USC").withStartDateAndTime("20-10-2021 09:00").withEndDateAndTime("20-10-2021 11:00")
+            .withAddress("USC").withStartDateAndTime("30-10-2021 09:00").withEndDateAndTime("30-10-2021 11:00")
```

Comment:
Why are you changing this though? You might need to change the typicalContactAndEventAddressBook

Reply:
It has the same start date as the VALID_START_DATE_TIME_EXAM, I thought it would be better if different events have different date and time range. Thanks for reminding to change it in typicalContactAndEventAddressBook :)